CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## 1.6.0 (2021.7.14)
+
+BACKWARDS COMPATIBILITY:
+* Deprecated the `distro.linux_distribution()` function. Use
+  `distro.id()`, `distro.version()` and `distro.name()` instead
+  [[#296](https://github.com/python-distro/distro/pull/296)]
+* Deprecated Python 2.7, 3.4 and 3.5 support. Further releases
+  will only support Python 3.6+ 
+
+ENHANCEMENTS:
+* Added type hints to `distro` module [[#269](https://github.com/python-distro/distro/pull/269)]
+* Added `__version__` for checking `distro` version [[#292](https://github.com/python-distro/distro/pull/292)]
+* Added support for arbitrary rootfs via the `root_dir` parameter [[#247](https://github.com/python-distro/distro/pull/247)]
+* Added the `--root-dir` option to CLI [[#161](https://github.com/python-distro/distro/issues/161)]
+* Added fallback to `/usr/lib/os-release` when `/etc/os-release` isn't available [[#262](https://github.com/python-distro/distro/pull/262)]
+
+BUG FIXES:
+* Fixed `subprocess.CalledProcessError` when running `lsb_release` [[#261](https://github.com/python-distro/distro/pull/261)]
+* Ignore `/etc/iredmail-release` file while parsing distribution [[#268](https://github.com/python-distro/distro/pull/268)]
+* Use a binary file for `/dev/null` to avoid `TextIOWrapper` overhead [[#271](https://github.com/python-distro/distro/pull/271)]
+
+RELEASE:
+* Moved repository from `nir0s/distro` to
+  [`python-distro/distro`](https://github.com/python-distro/distro) on GitHub.
+
+TESTS:
+* Changed CI to use GitHub Actions [[#277](https://github.com/python-distro/distro/pull/277)]
+
+DOCS:
+* Pinned intersphinx links to Python 3.7 [[#297](https://github.com/python-distro/distro/pull/297)]
```

Comment:
i'm not sure whether these are really needed respectively there are other changes like the introduction of black that seem similar to me, and imo aren't really important in this document to 'end users'. what about including [a hyperlink to the filtered commits](https://github.com/python-distro/distro/compare/v1.5.0...master) for this release?

Reply:
I'm not sure how I feel about filtered commits (especially since anyone interested can do this themselves in the GitHub UI) but I do agree that the docs/tests changes aren't relevant to end-users. What do you think?

Reply:
I agree with @sethmlarson . I don't think we need to provide the link. I prefer a curated set of relevant notes, as has been done here.